tests/Nox.Types.Tests/Types/VatNumberTests/VatNumberTests.cs

```diff
@@ -4,8 +4,107 @@ namespace Nox.Types.Tests.Types;
 public class VatNumberTests
 {
     [Fact]
-    public void When_Create_Should()
+    public void VatNumber_ParameterizedConstructor_Success()
     {
+        //Arrange
+        var vatNumberValue = "44403198682";
+        var countryCode = CountryCode2.From("FR");
 
+        //Act
+        var vatNumber = VatNumber.From(vatNumberValue, countryCode);
+
+        //Assert
+        Assert.Equal(vatNumberValue, vatNumber.VatNumberValue);
+        Assert.Equal(countryCode, vatNumber.CountryCode);
+        Assert.Equal("FR", vatNumber.CountryCode.ToString());
+    }
+
+    [Fact]
+    public void VatNumber_Should_Equal()
+    {
+        //Arrange
+        var vatNumber1 = VatNumber.From("44403198682", CountryCode2.From("FR"));
+        var vatNumber2 = VatNumber.From("44403198682", CountryCode2.From("FR"));
+
+        //Act
+        var result = vatNumber1.Equals(vatNumber2);
+
+        //Assert
+        Assert.True(result);
```

Comment:
See and use Fluent.Assertions, we are using for assertions

Reply:
Have updated tests to using FluentAssertions. One short question - is it a general rule for all the Nox libs to use just FluentAssertions for tests? I'm asking because I saw both Assert.True and FluentAssertions.

Reply:
In General,  new stuff with do it in Fluent, other parts we migrate whenever its possible